test/test_runner_test.rb

```diff
@@ -124,7 +124,7 @@ def test_syntax_errors_in_code
         end,A stray comma
            ^
       SYNTAX_ERRORS
-      assert_equal expected.strip, actual['message']
+      assert_equal expected, actual['message']
```

Comment:
What's the reason for these changes?

Reply:
Without that, 2 tests fail:

```
|   1) Failure:
| TestRunnerTest#test_syntax_errors_in_tests [/Users/katrinleinweber/Exercism.io/ruby-test-runner/test/test_runner_test.rb:111]:
| Expected: "Line 3: syntax error, unexpected ',', expecting end-of-input\n,'This is meant to be a syntax...\n^"
|   Actual: "Line 3: syntax error, unexpected ',', expecting end-of-input\n,'This is meant to be a syntax...\n^\n"
| 
|   2) Failure:
| TestRunnerTest#test_syntax_errors_in_code [/Users/katrinleinweber/Exercism.io/ruby-test-runner/test/test_runner_test.rb:127]:
| Expected: "Line 2: syntax error, unexpected ',', expecting end-of-input\nend,A stray comma\n   ^"
|   Actual: "Line 2: syntax error, unexpected ',', expecting end-of-input\nend,A stray comma\n   ^\n"
| 
| 12 runs, 19 assertions, 2 failures, 0 errors, 0 skips
```

I added an explanation to the description of https://github.com/exercism/ruby-test-runner/pull/47/commits/7cdb9402eb324baefa0fbd2e1aca6799481c8370 ;-)